packages/database/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "exclude": ["node_modules", "dist", "zod"],
+  "exclude": ["node_modules", "dist"],
   "extends": "@formbricks/config-typescript/node16.json",
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "tsup.config.ts"]
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "tsup.config.ts", "zod"]
```

Comment:
do we need the specific `zod` folder here since it should be covered by `"**/*.ts"`